test/unit/org/apache/cassandra/cql3/validation/operations/AlterTest.java

```diff
@@ -269,6 +276,63 @@ public void testCreateAlterKeyspaces() throws Throwable
                                   "max_threshold", "32")));
     }
 
+    @Test
+    public void testCreateAlterKeyspacesRFWarnings() throws Throwable
+    {
+        requireNetwork();
+        Thread.sleep(1000); // Hack: requireNetwork() will clear warnings async so give some time to complete
+
+        // NTS
+        ClientWarn.instance.captureWarnings();
+        execute("CREATE KEYSPACE testABC WITH replication = {'class' : 'NetworkTopologyStrategy', '" + DATA_CENTER + "' : 3 }");
```

Comment:
Not a big deal, but we could use [`CQLTester#createKeyspace`](https://github.com/apache/cassandra/blob/trunk/test/unit/org/apache/cassandra/cql3/CQLTester.java#L692) instead of `execute`, as it's done [here](https://github.com/adelapena/cassandra/commit/a42317a0e3ccf5d291b6629ee0c318298dcb994c) as an example. 

The advantage of `createKeyspace` is that it reduces the likelihood of collisions with other tests using the same keyspace names (IIRC we have had problems with this in the past). Also, `CQLTester` conditionally [drops the keyspaces](https://github.com/apache/cassandra/blob/trunk/test/unit/org/apache/cassandra/cql3/CQLTester.java#L344) created with `createKeyspace` after each test, so we wouldn't need the `DROP KEYSPACE` queries at the end of this tests. These drops will be executed even if the test fails, as if they were in a `finally` block, which is also an advantage.